api-metastore/src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -521,8 +534,43 @@ private void updateRetentionTime(final EventType original, final EventType event
     private void updateAnnotationsAndLabels(final EventType original, final EventType eventType)
             throws InvalidEventTypeException {
 
+       /* this is a temporary solution to ensure the backward compatibility and it is here because
+        the DataLakeAnnotationValidator.java does not have access to the event type */
+        final List<String> breakingCompatibilitiesNames = Arrays.asList(
```

Comment:
@adrianafirutanasie please remove the company-specific names from the public GitHub repository.